live-examples/js-examples/string/string-concat.html

```diff
@@ -0,0 +1,11 @@
+<pre>
+<code id="static-js">var str1 = 'Hello';
+var str2 = 'World';
+
+console.log(str1.concat(' ', str2));
+// expected output: 'Hello World'
```

Comment:
Very nit-picky, but expected output should exactly match the actual output, which here (and also below) means double quotes:

```
 // expected output: "Hello World" 
```

Sorry to be so nit-picky, but we have considered using "expected output:" for testing, and if we ever do that it will help for the content to match exactly.